polkadex-primitives/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+#![cfg_attr(all(not(target_env = "sgx"), not(feature = "std")), no_std)]
+#![cfg_attr(target_env = "sgx", feature(rustc_private))]
+
+#[cfg(feature = "sgx")]
+use sgx_tstd as std;
+
+use std::vec::Vec;
+
+use codec::{Encode, Decode};
+use sp_core::crypto::AccountId32;
+
+pub type AccountId = [u8;32];
```

Comment:
Same as in #22  - maybe use AccountId32?

Reply:
Hahable trait in not implemented for AccountId32 due to which we cant use it as key in Hashmap